src/sensors.c

```diff
@@ -46,7 +46,19 @@ void imu_ISR(void)
 
 static bool update_imu(void)
 {
-  if (_imu_ready)
+  if (_params.values[PARAM_HIL_ON])
+  {
+    // don't update normally, use HIL values
+    if(_imu_ready)
```

Comment:
So it looks like this still uses the timing from the MPU6050 interrupt rather than the timing from the incoming HIL IMU messages. Is that going to be a problem? Do we want to have the incoming IMU message event set a flag that `update_imu` checks instead of `_imu_ready` if the `PARAM_HIL_ON` param is set?